chromium_src/net/base/lookup_string_in_fixed_set.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define LookupSuffixInReversedSet LookupSuffixInReversedSet_ChromiumImpl
```

Comment:
Should #include `net/base/lookup_string_in_fixed_set.h` before redefining the symbol?

Reply:
Fixed in https://github.com/brave/brave-core/commit/b306d823968c0a6125b78908e8ee61082baa277e